src/Sum/README.md

```diff
@@ -0,0 +1,118 @@
+# Sum
+```haskell
+Sum Number
+```
+`Sum` is a `Monoid` that will combine two numbers using Addition.
+
+```js
+const Sum = require('crocks/Sum')
+const mconcat = require('crocks/helpers/mconcat')
+const mconcatMap = require('crocks/helpers/mconcatMap')
+
+Sum(13)
+//=> Sum 13
+
+mconcat(Sum, [5,2])
```

Comment:
Arrays should have spaces like:
```js
mconcat(Sum, [ 5, 2 ])
```

Reply:
Could you explain the reasoning for this?  For my own reasoning :)

Reply:
Many people, including me find the spacing easier to read when you have blocks of code.
Things tend to get missed when bit are all bungled.